charts/airbyte/templates/_helpers.tpl

```diff
@@ -154,61 +142,50 @@ Add environment variables to configure database values
 {{- printf "jdbc:postgresql://%s:%s/%s" $host $port $dbName -}}
 {{- end -}}
 
-{{/*
```

Comment:
what happened to these templates?

Reply:
Strange that PullSecrets got deleted, let me check on that.
About temporal, since I moved it into it's own chart, this template won't work as expected, I did it in much faster manner, simply by explicit editing of the configmap.yaml file. (New services will have following format: `{{ .Release.Name }}-{{ .Chart.Name }}` e.g `aribyte-worker`

Reply:
Reverted changes done to PullSecrets